.gitignore

```diff
@@ -2,3 +2,4 @@
 .git
 node_modules
 .idea
+.vs_code
```

Comment:
I thought the vs code directory was ``.vscode``?

Reply:
You're right. I actually merged the cookie cutter gitignore with this but haven't committed that work yet, so this is already fixed in my local.